src/term/color.rs

```diff
@@ -49,6 +49,10 @@ impl List {
         self[ansi::NamedColor::BrightMagenta] = colors.bright.magenta;
         self[ansi::NamedColor::BrightCyan]    = colors.bright.cyan;
         self[ansi::NamedColor::BrightWhite]   = colors.bright.white;
+        self[ansi::NamedColor::BrightForeground] = colors
+            .primary
+            .bright_foreground
+            .unwrap_or(colors.primary.foreground);
```

Comment:
Need to update the comment of the enclosing function to mention the new color(s).

Reply:
If you are talking about ![this](https://github.com/chrisduerr/alacritty/blob/aa639ce5e90054646942519e8e6db5a73e61eed0/src/term/color.rs#L9-L15), you're absolutely correct. Otherwise please specify exactly which comment you are talking about.

Reply:
You got it, sorry for not linking the exact lines.